### PR TITLE
Add NeuVector chart version 2.2.3

### PR DESCRIPTION
**Pull request by selvamt94**

### CODE REVIEW

File: packages/neuvector/package.yaml [hunk 1]
```diff
@@ -1,6 +1,5 @@
-url: https://neuvector.github.io/neuvector-helm/core-2.2.2.tgz
-version: 101.0.0
-doNotRelease: true
+url: https://neuvector.github.io/neuvector-helm/core-2.2.3.tgz
+version: 100.0.2
```
Comment (aiyengar2) [inline packages/neuvector/package.yaml:2]: dev-v2.7 should be incrementing the minor version by 1. See chart versioning guidelines in https://github.com/rancher/charts#versioning-charts.

The version in this PR should be `100.1.0`, which will come out as `100.1.0+up2.2.3` in `make charts`.

File: packages/neuvector/generated-changes/patch/Chart.yaml.patch [hunk 2]
```diff
@@ -10,13 +10,13 @@
 +  catalog.cattle.io/os: linux
 +  catalog.cattle.io/permit-os: linux
 +  catalog.cattle.io/provides-gvr: neuvector.com/v1
-+  catalog.cattle.io/rancher-version: '>= 2.6.5-0 < 2.7.0-0'
++  catalog.cattle.io/rancher-version: '>= 2.6.5-0 < 2.8.0-0'
```
Comment (aiyengar2) [inline packages/neuvector/generated-changes/patch/Chart.yaml.patch:13]: A 2.7 chart should only be scoped to 2.7. It should not be visible on 2.6.5.

```
'>= 2.7.0-0 < 2.8.0-0'
```

**Review by aiyengar2 — CHANGES_REQUESTED** [review]
Chart needs to be added to the `release.yaml`: https://github.com/rancher/charts/blob/dev-v2.7/release.yaml

See https://github.com/rancher/charts/blob/dev-v2.7/docs/validation.md#what-is-the-releaseyaml for explanation on why this exists.

You will need to add two entries: one for the CRD chart produced and one for the main chart. The versions should be something `100.1.0+up2.2.3`, as described in comments below.

File: packages/neuvector/package.yaml (hunk 1 above)
Comment (selvamt94) [reply packages/neuvector/package.yaml:2]: I will update the version to 100.1.0

File: packages/neuvector/generated-changes/patch/Chart.yaml.patch (hunk 2 above)
Comment (selvamt94) [reply packages/neuvector/generated-changes/patch/Chart.yaml.patch:13]: I will update it.

File: packages/neuvector/package.yaml [hunk 3]
```diff
@@ -1,6 +1,5 @@
-url: https://neuvector.github.io/neuvector-helm/core-2.2.2.tgz
-version: 101.0.0
-doNotRelease: true
+url: https://neuvector.github.io/neuvector-helm/core-2.2.3.tgz
+version: 100.1.0
```
Comment (aiyengar2) [inline packages/neuvector/package.yaml:2]: The chart should be released as 101.0.0 according to Rancher conventions in https://github.com/rancher/charts#versioning-charts

**Review by aiyengar2 — CHANGES_REQUESTED** [review]

**Comment by aiyengar2** [comment]
@selvamt94 apologies for the typo in my review! I should have listed 101.0.0 as a **major** version increment in https://github.com/rancher/charts/pull/2097#discussion_r978066172, not 100.1.0 as I had listed above.

File: packages/neuvector/package.yaml (hunk 1 above)
Comment (aiyengar2) [reply packages/neuvector/package.yaml]: Apologies for this! Should have been 101.0.0 as listed in the versioning guidelines I linked (but incorrectly quoted 😓)

File: packages/neuvector/package.yaml (hunk 3 above)
Comment (selvamt94) [reply packages/neuvector/package.yaml]: I have updated the version to 101.0.0. I will refer to the document before versioning the chart next time. Thanks.

**Comment by selvamt94** [comment]
> @selvamt94 apologies for the typo in my initial review! I should have listed 101.0.0 as a **major** version increment in [#2097 (comment)](https://github.com/rancher/charts/pull/2097#discussion_r978066172), not 100.1.0 as I had listed above.

No Problem

File: packages/neuvector/package.yaml (hunk 3 above)
Comment (aiyengar2) [reply packages/neuvector/package.yaml]: Thanks! and sorry once again for the confusion 😅

**Review by aiyengar2 — APPROVED** [review]
Thank you for making the changes! This PR looks good to me 😄

**Pull request merged**